fuzz.go

```diff
@@ -0,0 +1,45 @@
+//go:build gofuzz
+
+//go:generate go install github.com/dvyukov/go-fuzz/go-fuzz@latest
+//go:generate go install github.com/dvyukov/go-fuzz/go-fuzz-build@latest
+
+//go:generate go-fuzz-build
+//go:generate go-fuzz
+
+package dbresolver
+
+import (
+	"database/sql"
+	fuzz "github.com/google/gofuzz"
+)
+
+var LoadBalancerPolicies = []LoadBalancerPolicy{
+	RandomLB,
+	RoundRobinLB,
+}
+
+func Fuzz(data []byte) int {
+
+	var rdbCount, wdbCount, lbPolicyID uint8
+
+	fuzz.NewFromGoFuzz(data).Fuzz(&rdbCount)
+	fuzz.NewFromGoFuzz(data).Fuzz(&wdbCount)
+	fuzz.NewFromGoFuzz(data).Fuzz(&lbPolicyID)
+
```

Comment:
what is this ?

can we separate this into another PR, this is really not relevant.

Reply:
sure